schemas/ispyb/stored_programs/sp_retrieve_dc.sql

```diff
@@ -13,9 +13,11 @@ BEGIN
 					dc.detectorId "detectorId",
 					dc.blSubSampleId "blSubSampleId",
 					dc.dataCollectionNumber "dcNumber",
+					dc.APERTUREID "apertureId",
+					dc.POSITIONID "positionId",
 					dc.startTime "startTime",
 					dc.endTime "endTime",
-					dc.runStatus "status",
+					dc.runStatus "runStatus",
```

Comment:
Afraid we can't change the alias for a column in an existing procedure as it's already in use (e.g. by the Python API). I suggest you instead add the column again in the SELECT, but with the "runStatus" alias.

Reply:
Sure, I'll just change it on the Java side. It doesn't have to be called runStatus I just like to keep things on the descriptive side.